backend/app/rabbitmq/heartbeat_listener_2.py

```diff
@@ -0,0 +1,59 @@
+import pika
```

Comment:
sorry I might've missed 2 weeks of discussion. Is this meant for clowder 2? 
Any reason we are using `aio_pika` vs `pika` in different places?

Reply:
This is something I can change. Part of the issue I had was I was trying to run this as a background process, and I think running in using just pika caused issues.

If it's better, I can try to use pika again, especially since this will be running as a separate script outside of main.